admin/sql/create_tables.sql

```diff
@@ -67,7 +67,17 @@ CREATE TABLE revision (
     id          SERIAL      NOT NULL,
     review_id   UUID,
     "timestamp" TIMESTAMP   NOT NULL,
-    text        VARCHAR     NOT NULL
+    text        VARCHAR,     
+    rating      SMALLINT CHECK (rating >= 0 AND rating <= 100)
+);
+ALTER TABLE revision ADD CONSTRAINT revision_text_rating_both_not_null_together
+    CHECK (rating is NOT NULL OR text is NOT NULL);
+
+CREATE TABLE avg_rating (
+    entity_id   UUID         NOT NULL,
+    entity_type entity_types NOT NULL,
+    rating      SMALLINT     NOT NULL CHECK (rating >= 0 AND rating <= 100),
```

Comment:
It will be done as a part of db script. Every time, there's a revision with new rating, it will be updated.